west.yml

```diff
@@ -260,7 +260,7 @@ manifest:
       revision: ce57712f3e426bbbb13acaec97b45369f716f43a
       path: modules/lib/loramac-node
     - name: lvgl
-      revision: f7cf01d413aa9e76f376e25aa7b5c08dfdbc3c24
+      revision: pull/42/head
```

Comment:
5da257f782a8f9c6e265bdc60ebc2a93fdee24de all yours

Reply:
Update the revision :^)